scripts/uberenv/spack_configs/docker/ubuntu18/compilers.yaml

```diff
@@ -2,8 +2,7 @@ compilers:
   - compiler:
       environment: {}
       extra_rpaths: []
-      flags:
-        cxxflags: -stdlib=libc++
+      flags: {}
```

Comment:
This was the source of my Sol segfault.

Reply:
😢 At least you found it! Take a vacation day!